packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -0,0 +1,470 @@
+---
+title: MyAccount
+category: management
+---
+
+# MyAccount API
+
+<ApiLifecycle access="ea" />
+
+The Okta MyAccount API is a non-admin group of endpoints that allows end users (with or without administrator access) to fetch and update their own Okta user profiles, using a non-administrator endpoint.  It implements a subset of the existing [Users API](/docs/reference/users/index.md) but with significant differences.  This API does not expose information an end-user should not have access to, and it does not support lifecycle operations.
+
+All operations in this API implicitly refer to the user making the API call.  No user ID is needed (or even accepted).
+
+<!--
+## Get started
+
+Explore the MyAccount API:
+
+-->
+
+## MyAccount Operations
+
+The MyAccount API has the following operations:
+
+* [Get data about Me](#get-me)
+* [Get my MyUserSchema](#get-my-user-schema)
+* [Get my UserProfile](#get-my-user-profile)
+* [Update my UserProfile](#update-my-user-profile)
+
+### Get Me
+
+<ApiOperation method="get" url="/api/v1/myaccount" />
+
+Fetches the current user's Me object, a collection of links to information describing the user.
+
+#### Request path parameters
+
+N/A
+
+#### Request query parameters
+
+N/A
+
+#### Response body
+
+The requested [Me object](#me)
+
+> **Note:** Currently not all the `_links` returned are implemented.  See the [Me object](#me) description for details.
```

Comment:
If they aren't implemented, why are we returning them?

Reply:
Because that's what the spec says.

Reply:
Per Product we are TF'ing this out.  Removing references to phantom links.